### ISSUE TITLE
Brick Clamp (mass brick production)

### ISSUE THREAD

**Issue by kevingranade** [CleverRaven/Cataclysm-DDA#32581]
**Is your feature request related to a problem? Please describe.**  
When mass-producing bricks, there are no options for firing large numbers of bricks at once, especially in an unattended way.

**Describe the solution you'd like** 
Add a [brick clamp](http://www.fastonline.org/CD3WD_40/GATE_DL/BUILDING/BC/EN/BC.HTM) construction that can accept ~5,000 unfired bricks and a large amount of fuel and produce a large number of fired bricks.
Optionally:
Variants as outlined in the above document to improve yields.
Semi-attended production, e.g. adding more fuel over the course of the process (perhaps daily?) instead of loading it with fuel all at once, which is not how clamps operate.
Variable sizes of clamp based on how many bricks the player needs.

**Describe alternatives you've considered**  
Various kilns exist, but this is the simplest and most robust option, and it doesn't require excessive additional materials or construction.

**Comment by glenmack** [comment]
Substantially similar to #20420 which got no traction.

Some points - ~5,000 is village working together over a day or two, and is excessive for a single person. Even by the game standards to produce this many wet adobe bricks would take the survivor 62.5 hours of constant work to make this many adobe bricks.  I don't think many people are going to commit to a full work week and a half of batch crafting.

Also, Kevin's quoted article doesn't really state, but brick clamps are made of already dried bricks, as stated in [this article](https://www.benfleethistory.org.uk/content/browse-articles/buildings_and_development/brickworks/the_process_of_brick_making). So there's also drying time to consider. Also the space needed to allow that many bricks to dry in a climate where it rains quite often even during the summer.

This idea probably needs some kind of re-balance with wall construction, as it takes 30 bricks to make a wall, and 5000 bricks would pretty much be all the bricks needed to build all the buildings in the Lumber mill or an entire fire station. But it takes 4 hours to make each wall. so with 5000 bricks, that's 28 full continuous days of construction. WITHOUT counting making the mortar.

I think this is a case where real world numbers just SHOULDN'T apply to the game. I suggest you start with a satisfying number of walls to construct in a day, maybe 5-10. (an hour for each half of the wall?) Multiply that by number of bricks used in a day, and use that as a base number for the brick clamp. Base the crafting times around getting the brick clamp made with that many bricks done in 8 hours, with some kind of 16 hour drying or firing process.

In short, don't make it real if you're going to make "realism" the reason no one uses the feature.

**Comment by chronicpayne** [comment]
It does seem like the IRL advantages of brick do not transfer over well into CDDA at the current point in time.

Brick is one of the more durable building materials, but the main benefit of using it today is its longevity and lower maintenance vs similarly priced options, especially in areas where lumber is restricted or limited for whatever reason.  Additionally you see it for aesthetics, with faux brick being popular as well.

IMO None of these things factor into cddas construction very well, in a setting where the laws on tree harvesting are deader than your neighbours.

By all accounts Ive heard, bricks appear to fill a flavor niche and not much else.

**Comment by TychaiosIlithios** [comment]
Great idea, mine is a bit different. One of initial bonuses of cataclysm which make it far different from other survival games is the ability to mine the landscape for a lot of resources. Players who know it have far different situations from those who don't both very early and midgame.

So regarding bricks my idea is to make more of them obtainable by smashing walls. Irl I've done that a few times to old brick barns - result is either 70%ish return in blocks of 5-15 well held together bricks or 35-55% return of single bricks (depends on having a chisel or just dumb hammering). Such low return numbers are because the walls were not so new and the main goal was to deconstruct them relatively fast and not to spend time petting them. New wall sections made of such bricks looked ugly and had reliable durability only with double thickness, but with it - all was good enough.

So my full max idea is to have smashing brick walls result in brick rubble, which can be dug for 1-5 bricks and 1-3 brick blocks, brick blocks can be chiseled&hammered into 2-5 individual bricks each. 

Once again the main goal of this suggestion is to keep klizm advantages of cannibalizing environment effective and realistic.

**Comment by TechyBen** [comment]
Wait... do bricks not drop from smashing walls? I know of a few places around here *made* out of "reclaimed" bricks. Perhaps that could be an option allowing for a mix of both (adobe for when wilderness survival/adding to the crafting material needs, and reclaimed when requiring large number of crafting materials without additional time sink?).

https://youtu.be/hJ6Z731qxr8

Hammer and chisel. Time used cleaning/preparing the bricks (I guess if simming/modelling this mechanics, knocking a wall down would give a few "dirty bricks" with mortar on... using chisel and hammer on wall would give a few/more dirty bricks). Dirty bricks could be crafted into clean bricks with hammer + chisel again, but quicker than baking and forming entirely from new materials.

**Comment by kevingranade** [comment]
From @glenmack 's [source](https://www.benfleethistory.org.uk/content/browse-articles/buildings_and_development/brickworks/the_process_of_brick_making)
> If a brick gang of six individuals can produce on average of 900 bricks per hour, for an eight and a half-hour day, the number of bricks produced per day is 7,650.

IDK where the input number comes from, but let's just take it for the sake of argument.  that is indeed about a week of work to prep enough bricks for one clamp. I agree this isn't a sensible thing for a lone survivor to do (not that it'll stop them). And it's also true that you need to store and shelter the bricks for drying before even constructing the clamp.

> This idea probably needs some kind of re-balance with wall construction, as it takes 30 bricks to make a wall

Totally, 30 bricks for a 1 meter wall section is clearly wrong, for a 4m tall wall one meter in length and double thick (it NEEDS to be two rows, otherwise it's not at all secure), that's more like 400 bricks per wall section.

> But it takes 4 hours to make each wall. so with 5000 bricks, that's 28 full continuous days of construction.

Sounds about right, you need a team.

> I think this is a case where real world numbers just SHOULDN'T apply to the game.

If it doesn't make sense to build brick buildings, that's *totally fine*, then we just don't have any brick buildings, and instead we have buildings that actually make sense, maybe rammed earth or log timbers, who knows.  If the timelines are clearly non-representative of reality (I'm not getting a clear indication that there's a problem here, but there might be), then we need to figure out why and fix it, not paper over it by ignoring how long this stuff takes to do.

**Comment by nexusmrsep** [comment]
There are things not mentioned here as a possible source. Where would you got to get some? There would be shops (home depot) with construction materials, building companies with warehouses stocks, and actually brick making companies. Bricks wouldn't be a first need article in a disaster, so the stocks should be full.

**Comment by kevingranade** [comment]
Yea making your own is much more an innawoods activity, even if it's not the main game focus I like outlining what would be reasonable, in some cases it's more to establish a negative instead of add an option.
In this case manufacture is obviously and egregiously inefficient, so that should push us toward making other options available.

**Comment by brannamm** [comment]
I'd like to weigh in and say that even for a non-innawoods character that is just looking to make a robust base as a long-term goal, bricks (and the manufacture of bricks, even if it takes weeks) should be valuable. A brick base should be more defensible and safe than a wooden one. Several weeks of work to build a brick wall or even a brick "safe room" in your base seems very reasonable to me as long is there is tangible benefit to it, but as the game is right now I'm not sure if building one has much actual value. 

The discussion on how to make brick structures more useful probably doesn't belong here, but if bricks were a valuable commodity then I think the brick clamp as originally proposed (maybe some tweaks to numbers but keeping weeks of work for a whole brick building) is a good addition to the game.

**Comment by Zireael07** [comment]
We now have NPCs doing stuff and basecamps, and whatnot, they could help out with the brick making?
